astropy/cosmology/core.py

```diff
@@ -15,6 +15,9 @@
 from .connect import CosmologyFromFormat, CosmologyRead, CosmologyToFormat, CosmologyWrite
 from .parameter import Parameter
 
+if TYPE_CHECKING:
+    from astropy.cosmology.funcs.comparison import _FormatType
```

Comment:
I thought codecov was smart enough to skip these blocks because they are never run at runtime.

Reply:
Is there any harm in always importing `_FormatType`?

Reply:
Yes, but I think I've solved it by adding it to the ``coverage:report`` ``exclude_lines``, like ``# pragma: no cover``, so it's never counted towards coverage.

Reply:
🤞

Reply:
@pllim do you know if changes to ``[coverage:report]`` in ``setup.cfg`` affect the PR in which it's changed?
I've modified the ``setup.cfg`` according to a number of example PRs. Not sure what's going on.

Reply:
Yes, I think coverage grabs the `setup.cfg` that is shipped with the PR. Can you just use `# pragma: no cover` on that line above instead of modifying `setup.cfg`?

Reply:
It looks like skipping ``if TYPE_CHECKING`` might become the default for ``coverage.py``, but I'm not sure when that'll trickle down to us.